src/windows/views/add_to_timeline_treeview.py

```diff
@@ -77,8 +77,8 @@ def __init__(self, *args):
         self.setModel(self.timeline_model.model)
         self.setIconSize(info.TREE_ICON_SIZE)
         self.setIndentation(0)
-        self.setSelectionBehavior(QTreeView.SelectRows)
-        self.setSelectionBehavior(QAbstractItemView.SelectRows)
+        self.setSelectionBehavior(QAbstractItemView.SelectionBehavior.SelectRows)
```

Comment:
This line looks like a duplicate

Reply:
Good call, my regular-expression-based updates even turned them into identical duplicates, one can clearly go.